manifests/init.pp

```diff
@@ -347,6 +347,7 @@
 # @param service_hasstatus
 # @param service_hasrestart
 # @param acknowledge_unsupported_os
+# @param keyring
```

Comment:
Should add documentation, not just list it here.

Reply:
```suggestion
# @param keyring
#   Absolute path to a file containing the PGP keyring used to sign this repository. Value is used to set signed-by on the source entry.
#   See https://wiki.debian.org/DebianRepository/UseThirdParty for details.
```
Add description, copied from puppetlabs-apt.